XiEditor/EditViewController.swift

```diff
@@ -158,7 +158,15 @@ class EditViewController: NSViewController, EditViewDataSource {
         lastDragPosition = position
         let flags = theEvent.modifierFlags.rawValue >> 16
         let clickCount = theEvent.clickCount
-        document.sendRpcAsync("click", params: [position.line, position.column, flags, clickCount])
+        if theEvent.modifierFlags.contains(NSCommandKeyMask) {
```

Comment:
this is also capturing any combination of modifier flags that _includes_ command, e.g. ctrl+shift+command. Is that desired behaviour?